src/utils/httpVariableProviders/environmentVariableProvider.ts

```diff
@@ -44,11 +53,25 @@ export class EnvironmentVariableProvider implements HttpVariableProvider {
         return Object.keys(variables).map(key => ({ name: key, value: variables[key]}));
```

Comment:
`GetAll` method should also return the actual value for referenced variable as `Get`